.github/pull_request_template.md

```diff
@@ -0,0 +1,10 @@
+## Describe your changes
+
+## Issue ticket number and link (if applicable)
+
+## Checklist before requesting a review
```

Comment:
Should we add `documentation written`  (if necessary/applicable) on the checklist? If a test is written or code updated, I think some documentation should be updated with it.

Reply:
good catch, addressing.